### DISCUSSION TITLE
Need steps to configure SEB for Unmanaged Device

### DISCUSSION THREAD

**Discussion by toanjan05** [SafeExamBrowser/seb-win-refactoring#379] [Q&A]
Hi, 

Could you please let me know the steps to configure SEB for unmanaged device? The objective is that the student can download the SEB  from the portal and then he/she can launch the test via SEB in our portal.

**Comment by dbuechel** [comment]
Please have a look at the documentation, it should provide all the basic information you'll need to get started: https://safeexambrowser.org/windows/win_usermanual_en.html. For your specific use case, there's furthermore an article on our website which should explain everything in detail, see https://safeexambrowser.org/developer/seb-integration.html.

**Reply by toanjan05** [reply]
Thanks for your answer. I need one clarification regarding the location of seb configuration file on unmanaged device.  As per documentation, it has been asked to place the configuration file either "%PROGRAMDATA%\SafeExamBrowser\SebClientSettings.seb" or "%APPDATA%\SafeExamBrowser\SebClientSettings.seb" location for the managed device. Is it required to keep the configuration file in the same location for the unmanaged device as well? Or can we keep the file anywhere in the system ?

**Reply by dbuechel** [reply]
If you'd like that SEB automatically uses that local configuration file when starting up, then yes, you'll need to place it in the program or app data folder (the former if you want all users on that system to use the same file, the latter if you need a different file per user).
